aea/aea_builder.py

```diff
@@ -780,7 +780,7 @@ def _load_and_add_contracts(self) -> None:
 
     def _load_and_add_skills(self, context: AgentContext) -> None:
         for configuration in self._package_dependency_manager.skills.values():
-            logger_name = "aea.{}.skills.{}.{}".format(
+            logger_name = "aea.packages.{}.{}.skills.{}".format(
```

Comment:
This looks wrong.

Reply:
should be `"aea.packages.{}.skills.{}.{}"` ? where first is author, second is skill name, third is module name?

Reply:
yes, sorry.

Actually, it should be `aea.{agent_name}.packages.{package_author}.skills.{skill_name}`